fury/tests/test_actors.py

```diff
@@ -310,6 +310,59 @@ def test_contour_from_roi():
         # window.show(r2)
 
 
+def test_contour_from_label():
+
+    # Render volumne
+    scene = window.Scene()
+    data = np.zeros((50, 50, 50))
+    data[15:20, 25, 25] = 1.
+    data[25, 20:30, 25] = 2.
+    data[25, 40:50, 30:50] = 3.
+
+    color = np.array(
+        [[1, 0, 0, 0.6],
+         [0, 1, 0, 0.5],
+         [0, 0, 1, 1.0]])
+
+    surface = actor.contour_from_label(data, color=color)
+
+    scene.add(surface)
+    scene.reset_camera()
+    scene.reset_clipping_range()
+    # window.show(scene)
+
+    # Test Errors
+    with npt.assert_raises(ValueError):
+        actor.contour_from_label(data, color=np.array([1, 2, 3]))
+        actor.contour_from_label(np.ones(50))
+
+    # Test binarization
+    scene2 = window.Scene()
+    data2 = np.zeros((50, 50, 50))
+    data2[20:30, 25, 25] = 1.
+    data2[25, 20:30, 25] = 2.
+
+    color2 = np.array(
+        [[1, 0, 1],
+         [1, 1, 0]])
+
+    surface2 = actor.contour_from_label(data2, color=color2)
+
+    scene2.add(surface2)
+    scene2.reset_camera()
+    scene2.reset_clipping_range()
+    # window.show(scene2)
+
+    arr = window.snapshot(scene, 'test_surface.png', offscreen=True)
```

Comment:
since you are using alpha channel (opacity). try to add this option here: `order_transparent=True`. Be careful with pep8 when adding this option. I hope it will solve the masOS issue.

Reply:
Did the required changes.